packet/flatcar-linux/kubernetes/calico/host-protection.yaml.tmpl

```diff
@@ -1,126 +1,109 @@
     destination:
       ports: [22]
-  # DHCP access
-  - action: Allow
-    protocol: UDP
-    source:
-      nets: ${node_public_ips}
-    destination:
-      ports: [68]
-  # BGP and etcd access
-  - action: Allow
-    protocol: TCP
-    source:
-      nets: ${node_public_ips}
-    destination:
-      ports:
-      # BGP access (Calico networking)
-      - 179
-      # etcd access
-      - 2379
-      - 2380
-      # etcd self-hosted service access
-      - 6666
-      - 6667
 ---
-# Allow traffic from IP addresses that are known to be cluster-internal including pods and services
+# kubectl access
 apiVersion: crd.projectcalico.org/v1
 kind: GlobalNetworkPolicy
 metadata:
-  name: allow-cluster-internal-ingress
+  name: kubectl
 spec:
-  order: 10
+  selector: nodetype == 'controller'
   preDNAT: true
   applyOnForward: true
+  order: 0
   ingress:
-    - action: Allow
-      source:
-        nets: ${cluster_internal_cidrs}
-  selector: has(host-endpoint)
+  - action: Allow
+    protocol: TCP
+    destination:
+      ports: [6443]
 ---
-# Deny traffic from any other sources
+# Traffic between cluster nodes
 apiVersion: crd.projectcalico.org/v1
 kind: GlobalNetworkPolicy
 metadata:
-  name: drop-other-ingress
+  name: inter-node-traffic
 spec:
-  order: 20
+  selector: has(host-endpoint)
   preDNAT: true
   applyOnForward: true
-  ingress:
-    - action: Deny
-  selector: has(host-endpoint)
----
-# Allow egress traffic through nodes' external interface
-apiVersion: crd.projectcalico.org/v1
-kind: GlobalNetworkPolicy
-metadata:
-  name: allow-outbound-external
-spec:
   order: 10
-  egress:
+  ingress:
     - action: Allow
-  selector: has(host-endpoint)
----
-# controller hosts endpoint
-${controller_host_endpoints}
----
-# worker hosts endpoint
-${worker_host_endpoints}
+      source:
+        nets: ${cluster_internal_cidrs}
 ---
-# api server access from public
+# Ingress traffic to Contour
 apiVersion: crd.projectcalico.org/v1
 kind: GlobalNetworkPolicy
 metadata:
-  name: allow-nodeport-apiserver
+  name: contour-ingress
 spec:
+  selector: nodetype == 'worker'
   preDNAT: true
   applyOnForward: true
-  order: 10
+  order: 20
   ingress:
   - action: Allow
     protocol: TCP
     destination:
-      ports: [6443]
-  selector: nodetype == 'controller'
+      ports: [80, 443]
 ---
-# Allow incoming traffic to Contour
+# Egress traffic
 apiVersion: crd.projectcalico.org/v1
 kind: GlobalNetworkPolicy
 metadata:
-  name: contour-ingress
+  name: egress
 spec:
-  preDNAT: true
-  applyOnForward: true
+  selector: has(host-endpoint)
   order: 0
-  ingress:
-  - action: Allow
-    protocol: TCP
-    destination:
-      ports: [80, 443]
-  selector: nodetype == 'worker'
+  egress:
+    - action: Allow
 ---
-# Disable default failsafe inbound rules
+# Cleanup - deny any ingress traffic not explicitly permitted
 apiVersion: crd.projectcalico.org/v1
-kind: FelixConfiguration
+kind: GlobalNetworkPolicy
 metadata:
-  name: default
+  name: cleanup
```

Comment:
cleanup? It seems like a weird name to see in the resources. I'm no good at namings, though, but what about `catch-all`, `default-policy` or something else?

Reply:
Yeah, `cleanup` doesn't make sense sense for me here. How about something like `deny-all`? Or `default-deny-all-ingress`? To make then name of each policy descriptive.

Reply:
A "cleanup rule" is a common practice in firewalls. It means "the very last rule which ensures anything not explicitly permitted by a firewall is dropped".

I can rename it though as it seems this term is confusing people.

Reply:
Renamed to `deny-other`.